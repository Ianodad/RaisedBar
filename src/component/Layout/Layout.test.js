import React from 'react';
import ReactDOM from 'react-dom';

import index from './index';

import { render, fireEvent, cleanup } from "@testing-library/react";
import renderer from 'react-test-renderer'

describe("BaseLayoutComponent", () => {
  // Start snapshot test
  it("<Layout/> renders in any case", () => {
    const tree = renderer.create(<index />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});