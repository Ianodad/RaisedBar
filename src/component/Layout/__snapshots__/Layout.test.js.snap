// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseLayoutComponent <Layout/> renders in any case 1`] = `<index />`;
